nd4j/nd4j-backends/nd4j-api-parent/nd4j-api/src/main/java/org/nd4j/linalg/api/ops/impl/transforms/same/Reciprocal.java

```diff
@@ -30,13 +31,15 @@
 /**
  * Created by susaneraly on 3/28/18.
  */
+@NoArgsConstructor
 public class Reciprocal extends BaseTransformSameOp {
 
-    public Reciprocal(SameDiff sameDiff, SDVariable in, boolean inPlace) {
-        super(sameDiff, in, inPlace);
+    public Reciprocal(SameDiff sameDiff, SDVariable in) {
+        this(sameDiff, in, false);
     }
 
-    public Reciprocal() {
+    public Reciprocal(SameDiff sameDiff, SDVariable in, boolean inPlace) {
+        super(sameDiff, in, inPlace);
     }
```

Comment:
inPlace = false always for SameDiff (don't want SameDiff constructor with boolean inPlace arg)

Looks like a bunch of others later (mostly pre-existing though)